packages/flutter/lib/src/material/app.dart

```diff
@@ -52,6 +52,7 @@ class MaterialApp extends StatefulWidget {
     this.theme,
     this.home,
     this.routes: const <String, WidgetBuilder>{},
+    this.initialRoute,
```

Comment:
Would it more sense to take a `Navigator` instead of a subset of `Navigator`'s attributes?